index.bs

```diff
@@ -242,7 +245,7 @@ initiateMultiScreenExperienceButton.addEventListener('click', async () => {
 # Concepts # {#concepts}
 <!-- ====================================================================== -->
 
-Concepts in this specification build upon those in [[CSSOM-VIEW]], the [CSSOM-View Editor's draft](https://drafts.csswg.org/cssom-view/) dated "15 November 2021", [[HTML]], and [[Fullscreen]].
+Concepts in this specification build upon those in [[CSSOM-VIEW-1]], the [CSSOM-View-1 Editor's draft](https://drafts.csswg.org/cssom-view/) dated "15 November 2021", [[HTML]], and [[Fullscreen]].
```

Comment:
Fine with me. The reason I added `-1` was to avoid ending up with two CSSOM-VIEW entries in the list of references (CSSOM-VIEW and CSSOM-VIEW-1) that targeted the same spec.